dateparser_cli/cli.py

```diff
@@ -0,0 +1,25 @@
+import sys
+from .fasttext_manager import fasttext_downloader
+from .utils import clear_cache
+
+
+_cli_functions_map = {
+    "fasttext": fasttext_downloader,
+    "clear_cache": clear_cache
+}
+
+
+def no_matching_command_found():
+    print("dateparser-download: No command found")
```

Comment:
Also I think we should raise an error in this case or set return code of the script to non-zero. Another option is to use ``argparse`` library to do the argument parsing and validation instead of doing that ourselves.

Reply:
I have created a custom error class and raised the same